extensions/data-plane/data-plane-http/src/main/java/org/eclipse/edc/connector/dataplane/http/pipeline/HttpDataSinkFactory.java

```diff
@@ -61,12 +61,21 @@ public boolean canHandle(DataFlowRequest request) {
 
     @Override
     public @NotNull Result<Boolean> validate(DataFlowRequest request) {
+        if (validateRequest(request).succeeded()) {
+            return VALID;
+        } else {
+            return Result.failure("The validation failed ");
+        }
```

Comment:
I think this could be simplified to:
`return validateRequest(request).map(it -> true);`